### PR TITLE
added styling for new footer

### PR DESCRIPTION
**Pull request by andylushman**
Footer update with `Stay In Touch` section

### CODE REVIEW

File: app/views/layouts/shared/_footer.html.slim [hunk 1]
```diff
@@ -4,16 +4,20 @@ footer.Footer
...
 
     .Footer-block
-      div.Footer-title Denver Startup Week #{Date.today.year}
-      div.Footer-subtitle &copy;Copyright #{Date.today.year}. All rights reserved.
-      = render 'components/social_media_nav'
\ No newline at end of file
+      .Footer-updates 
+        p.Footer-updates-title Stay in Touch
+        = form_for NewsletterSignup.new, class: "footer-updates-form", html: { data: { bindable: "newsletter-signup" } } do |f|
+          = f.text_field :email, placeholder: "Email Address", class: "Footer-updates-email"
```
Comment (scpaye506) [inline app/views/layouts/shared/_footer.html.slim]: you could use the `floating_input` component here. 

```
= render layout: 'components/floating_input' do
  = f.text_field ....
  = f.label ...
```
label has to go after the text_field with how it's currently setup

File: app/views/layouts/shared/_footer.html.slim [hunk 2]
```diff
@@ -4,16 +4,20 @@ footer.Footer
...
+      .Footer-title Denver Startup Week #{Date.today.year}
+      .Footer-subtitle &copy;Copyright #{Date.today.year}. All rights reserved.
+      = render 'components/social_media_nav'
+      ul.Footer-page-links-list aria-label="Footer page links"
         - if signed_in? && current_user.is_admin?
           li.Footer-link= link_to 'Admin', admin_root_path
-        li.Footer-link= link_to 'Attendees', page_path(page: 'attendees')
-        li.Footer-link= link_to 'Podcast', page_path(page: 'podcast')
-        li.Footer-link= link_to 'Press', page_path(page: 'press')
+        li.Footer-link= link_to 'FAQ', page_path(page: 'about/faq')
```
Comment (scpaye506) [inline app/views/layouts/shared/_footer.html.slim]: thoughts on putting the footer routes in the `route_helper`?

File: app/assets/stylesheets/redesign/views/footers.sass [hunk 3]
```diff
@@ -44,6 +50,31 @@
...
+  margin-bottom: 31px
+  
+.Footer-updates-email
+  +font-family-mixin(Montserrat-Light)
+  font-size: 14px
+  border-bottom: 1px solid $primary-font-light
+
+.Footer-updates-submit
+  +white-button(166px, 46px)
+  width: fit-content
```
Comment (scpaye506) [inline app/assets/stylesheets/redesign/views/footers.sass]: i'm not really sure we should be using `fit-content` it's not actually part of the spec and not supported by all browsers
https://developer.mozilla.org/en-US/docs/Web/CSS/width

If the `white-button` mixin is causing problems we should probably refactor that to work instead or just sit a fixed width where needed

File: app/views/layouts/shared/_footer.html.slim (hunk 1 above)
Comment (andylushman) [reply app/views/layouts/shared/_footer.html.slim]: Nice!

File: app/views/layouts/shared/_footer_nav.html.slim [hunk 4]
```diff
@@ -0,0 +1,7 @@
+nav.FooterNav(aria-label="#{nav_title} nav")
+  ul.FooterNav-page-links-list
+    - if signed_in? && current_user.is_admin?
```
Comment (scpaye506) [inline app/views/layouts/shared/_footer_nav.html.slim]: I think we might want to move this logic to the `router_helper`, otherwise looks good to go

Comment (andylushman) [reply app/views/layouts/shared/_footer_nav.html.slim]: 👍 Just moved it over.

**Pull request merged**